vignettes/customise.Rmd

```diff
@@ -107,21 +123,47 @@ template:
     code_font: {google: "JetBrains Mono"}
 ```
 
-While iterating on colours and other variables you only need to rerun `init_site()` and refresh your browser; when iterating on fonts, you'll need to run `build_home_index(); init_site()`.
+If you want to use a non-Google font, you'll need to do a bit more work. There are two steps: you need to first configure the font with CSS and then use it in your `_pkgdown.yaml`. There are two ways you might get the CSS:
+
+* As a block of CSS which you should put in `pkgdown/extra.scss` or `pkgdown/extra.css`. The CSS will look something like this:
+
+    ```css
+    @font-face {
+      font-family: "proxima-nova";
+      src: 
+        local("Proxima Nova Regular"), 
+        local("ProximaNova-Regular"),
+        url("https://example.com/ProximaNova-Regular.eot?#iefix") format("embedded-opentype"),
+        url("https://example.com/fonts/proxima/ProximaNova-Regular.woff2") format("woff2"),
+        url("https://example.com/fonts/proxima/ProximaNova-Regular.woff") format("woff"),
+        url("https://example.com/fonts/proxima/ProximaNova-Regular.ttf") format("truetype");
+      font-weight: normal;
+      font-style: normal;
+      font-display: fallback;
+    }
+    ```
+
+* As a link to a style file, which you'll need to add to the `<head>` using this syntax:
 
-Theming with bslib is powered by `bslib::bs_theme()` and the `bslib` field is a direct translation of the arguments to that function.
-As a result, you can fully specify a bslib theme using the `template.bslib` field, making it easy to share YAML with the `output.html_document.theme` field [of an R Markdown document](https://rstudio.github.io/bslib/articles/theming/index.html).
+    ```yaml
+    template:
+    includes:
+      in_header: <link rel="stylesheet" type="text/css" href="https://..." />
+    ```
 
-``` yaml
+Then in `_pkgdown.yml` you can use the name of the font you just specified:
+
+```yaml
 template:
   bslib:
-    version: 5
-    bootswatch: lux
-    base_font: {google: "Roboto"}
-    heading_font: {google: "Roboto Slab"}
-    code_font: {google: "JetBrains Mono"}
+    base_font: proxima-nova
 ```
 
+Depending on where the font is from (and if you paid money for it), you may need to take additional steps to ensure that it can only be used from your site, and/or make sure that it can still be used when you're previewing locally. If you're having problems getting a custom font to work, looking for errors in the browser developer console is a good place to start.
```

Comment:
```suggestion
Depending on where the font is from (and if you paid money for it), you may need to take additional steps to ensure that it can only be used from your site, and/or make sure that it can still be used when you're previewing locally. If you're having problems getting a custom font to work, looking for errors in the [browser developer console](https://developer.mozilla.org/en-US/docs/Glossary/Developer_Tools) is a good place to start.
```